docs/quick-start.md

```diff
@@ -376,15 +373,175 @@ curl -i --url http://127.0.0.1:8428/api/v1/query --url-query 'query=demo_counter
+  jq -r '.data.groups[].rules[].name';
+# DemoAlertFiring
+```
+
+If you prefer using a web interface, you can open the `VMAlert` UI in your browser.
+First, forward port `8080` from the `VMAlert` pod to your local machine:
+```sh
+VMALERT_POD_NAME=$(kubectl get pod -n vm -l "app.kubernetes.io/name=vmalert" -o jsonpath="{.items[0].metadata.name}")
+kubectl port-forward -n vm $VMALERT_POD_NAME 8080:8080
+```
+Then go to: http://localhost:8080/vmalert/groups.
+You’ll see all groups, alerts and a lot more.
+
+That’s it! Your alerting setup is now working.
+If you want to see how it works from start to finish, follow the steps below.
+
+To trigger an alert, run this command:
+```sh
+DEMO_APP_POD=$(kubectl get pod -n default -l "app.kubernetes.io/name=demo-app" -o jsonpath="{.items[0].metadata.name}");
+kubectl exec -n default ${DEMO_APP_POD} -- curl -s --url http://127.0.0.1:8080/alerting/fireDemoAlert;
+kubectl exec -n default ${DEMO_APP_POD} -- curl -s --url http://127.0.0.1:8080/metrics;
+
+# demo_alert_firing 1
+# demo_counter_total 523
+```
+
+This sets the `demo_alert_firing` metric to 1. Give alert a couple of minutes to fire, then check the webhook receiver:
+```sh
+DEMO_APP_POD=$(kubectl get pod -n default -l "app.kubernetes.io/name=demo-app" -o jsonpath="{.items[0].metadata.name}");
+kubectl exec -n default ${DEMO_APP_POD} -- curl -s --url http://127.0.0.1:8080/alerting/receivedWebhooks;
+
+# {"receiver":"demo-app","status":"firing","alerts":[{"status":"firing","labels":{"alertgroup":"demo-app","alertname":"DemoAlertFiring","job":"demo-app","namespace":"default","pod":"demo-app-7f65f4dbf7-kt4tz"},"annotations":{"description":"demo-app pod demo-app-7f65f4dbf7-kt4tz is firing demo alert"},"startsAt":"2025-05-09T15:07:40Z","endsAt":"0001-01-01T00:00:00Z","generatorURL":"http://vmalert-demo-6f9cfcfb54-zzz9d:8080/vmalert/alert?group_id=1974157196182235209\u0026alert_id=9826110110139929675","fingerprint":"d014d7d794d8b310"}],"groupLabels":{},"commonLabels":{"alertgroup":"demo-app","alertname":"DemoAlertFiring","job":"demo-app","namespace":"default","pod":"demo-app-7f65f4dbf7-kt4tz"},"commonAnnotations":{"description":"demo-app pod demo-app-7f65f4dbf7-kt4tz is firing demo alert"},"externalURL":"http://vmalertmanager-demo-0:9093","version":"4","groupKey":"{}:{}","truncatedAlerts":0}
+```
+It will output the most recent webhook received from `VMAlertmanager`.
+
+To stop the alert, run:
+```sh
+DEMO_APP_POD=$(kubectl get pod -n default -l "app.kubernetes.io/name=demo-app" -o jsonpath="{.items[0].metadata.name}");
+kubectl exec -n default ${DEMO_APP_POD} -- curl -s --url http://127.0.0.1:8080/alerting/resolveDemoAlert;
+kubectl exec -n default ${DEMO_APP_POD} -- curl -s --url http://127.0.0.1:8080/metrics;
+
+# demo_alert_firing 0
+# demo_counter_total 601
+```
+This sets the `demo_alert_firing` metric to 0. You should stop receiving alert notifications. 
```

Comment:
>You should stop receiving alert notifications.

You should receive one final alert resolved notification.

Reply:
fixed